src/Compiler/Compiler.cpp

```diff
@@ -1335,8 +1335,7 @@ int Compiler::ExecuteAndMonitorSystemCommand(const std::string& command) {
   (*m_out) << "Command: " << command << std::endl;
   auto path = std::filesystem::current_path();  // getting path
   (*m_out) << "Path: " << path.string() << std::endl;
-  std::filesystem::current_path(path /
-                                m_projManager->projectName());  // setting path
+  std::filesystem::current_path(m_projManager->projectPath());  // setting path
```

Comment:
Why was this a problem?

Reply:
this will not work when you open project.

Reply:
Also doesn't work when project created in not default place.